### PR TITLE
Feature create sport field

### LINKED ISSUE #4: Owner user can create a sports field
- Description: 
**As an** owner user, 
**I can** create a new sports field by specifying the name, description, sports kind, photo and price for day and night, 
**so that** I can manage it.

<div align="center">
CreateSportsField - Owner
</div>

### PR DESCRIPTION
**Pull request by condef5**
### Status
Required

### Implementation Notes
* Add sport_field model and migration
* Add routes and method create in sport_fields_controller
* Add test for sport_fields_controller
* Add serializer for sport_field model
* Add view for create-sport-field
* Add test for create-sport-field

closed #4

### CODE REVIEW

**Comment by cristianbgp** [comment]
Remember to add the Issue you are closing in the description

File: client/src/services/sport-field.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import fetch from "isomorphic-fetch";
```
Comment (liannivin) [inline client/src/services/sport-field.js]: Could you please explain what are you trying to do with this?

File: client/src/views/create-sport-field.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            placeholder="Sports field's name"
+            value={fields.name}
+            onChange={handleChange}
+          />
+        </div>
+        <div css={{ marginTop: "2em" }}>
+          <Label htmlFor="description">Description</Label>
+          <Input
+            aria-label="enter description"
+            required="required"
```
Comment (liannivin) [inline client/src/views/create-sport-field.js]: Could you please explain what are you trying do with this. Normally `required` doesn't have a value. Check this: https://www.geeksforgeeks.org/html-required-attribute/

File: client/src/services/sport-field.js (hunk 1 above)
Comment (condef5) [reply client/src/services/sport-field.js]: Sorry, I tried in another branch mock the asynchronous petitions, now I fix it.

File: client/src/views/create-sport-field.js (hunk 2 above)
Comment (condef5) [reply client/src/views/create-sport-field.js]: Of course, now I fix it, thanks :)

Comment (liannivin) [reply client/src/views/create-sport-field.js]: Thanks @condef5. Take care with `english` grammar: `fixed` instead `fix`.

**Pull request merged**